graph/config/cytoscape/cytoscape.go

```diff
@@ -251,13 +255,7 @@ func buildConfig(trafficMap graph.TrafficMap, nodes *[]*NodeWrapper, edges *[]*E
 
 		addNodeTelemetry(n, nd)
 
-		// set annotations, if available
-		if val, ok := n.Metadata[graph.HealthData]; ok {
-			nd.HealthData = val.(interface{})
-		}
-		if val, ok := n.Metadata[graph.HealthDataApp]; ok {
-			nd.HealthDataApp = val.(interface{})
-		}
```

Comment:
@aljesusg this could be the cause of the frontend failures from CI. You'll likely need to keep the assignment but maybe remove the type cast to `interface{}` to make the linter happy.